### ISSUE TITLE
In Android Studio 2.3, `Autoconnect` only works with parents

### ISSUE THREAD

**Issue by dannyroa** [googlecodelabs/constraint-layout#21]
The codelabs is incorrect because instruction is to use `AutoConnect` to neighboring views.

**Comment by zawmai** [comment]
To expand on the issue, look at android developer reference [Build a Responsive UI with ConstraintLayout](https://developer.android.com/training/constraint-layout/index.html#use-autoconnect-and-infer-constraints).

Under section "Automatically Create Constraints", it says the following: 

> When turned on, it [autoconnect] automatically creates two or more constraints for each view as you add them to the layout, but only when appropriate to constrain the view to the parent layout. Autoconnect does not create constraints to other views in the layout.

tl;dr: Autoconnect only make constraints with respect to parent layout, not other views.

**Comment by nageshs** [comment]
Thanks. Yes the AutoConnect behavior has been modified. Let me update the codelab!

**Comment by jeancdc** [comment]
+1.
I try the ConstraintLayout today and I see that the Autoconnect only work with the parents.

**Comment by micer** [comment]
+1
Today I couln't finish step 8 of codelabs because of this as well. The tutorial should be updated. Thanks @nageshs!

**Comment by sonichanxiao** [comment]
I couldn't find the AutoConnect button/icon mentioned. 
Could anyone remind me where it is, please?

**Comment by zawmai** [comment]
@dannyroa I think this issue should be closed now.
